scripts/asm-installer/install_asm

```diff
@@ -2491,15 +2494,15 @@ configure_package() {
 
   if [[ "${CA}" == "mesh_ca" ]]; then
     # set the trust domain aliases to include both new Hub WIP and old Hub WIP to achieve no downtime upgrade.
-    TRUST_DOMAIN_ALIASES="${PROJECT_ID}.svc.id.goog ${PROJECT_ID}.hub.id.goog"
+    TRUST_DOMAIN_ALIASES="${PROJECT_ID}.svc.id.goog ${PROJECT_ID}.hub.id.goog ${TRUST_DOMAIN_ALIASES}"
     run eval kpt cfg set asm anthos.servicemesh.trustDomainAliases "${TRUST_DOMAIN_ALIASES}"
   fi
 
   if [[ "${CA}" == "mesh_ca" && -n "${_CI_TRUSTED_GCP_PROJECTS}" ]]; then
     # Gather the trust domain aliases from projects.
-    TRUST_DOMAIN_ALIASES="${PROJECT_ID}.svc.id.goog"
+    TRUST_DOMAIN_ALIASES="${PROJECT_ID}.svc.id.goog ${TRUST_DOMAIN_ALIASES}"
```

Comment:
Do we append the `${PROJECT_ID}.svc.id.goog twice` if TRUST_DOMAIN_ALIASES is appended in the if [[ "${CA}" == "mesh_ca" ]] and ten we append one more time here?

Reply:
depends on whether the previous logic is triggered.. that issue might predate this PR